src/Components/Reports/Common/ExecutiveFirstPage.js

```diff
@@ -2,20 +2,19 @@ import React, { Fragment } from 'react';
 import propTypes from 'prop-types';
 import { Section, Column, Chart, Table } from '@redhat-cloud-services/frontend-components-pdf-generator';
 import { Text, View } from '@react-pdf/renderer';
-import { formatExecutiveReportData } from '../Util';
+import { formatExecutiveReportData, pluralize } from '../Util';
 import styles from './styles';
 import { IconCanvas } from './IconCanvas';
 
 export const ExecutiveFirstPage = ({ data }) => {
-    const { conditions_count: conditionsCount, non_optimized_count: nonOptimizedCount, total_count: totalCount } = data?.meta;
+    const { conditions_count: conditionsCount, non_optimized_count: nonOptimizedCount,
+        total_count: totalCount, stale_count: staleCount } = data?.meta;
     const optimizedCount = data?.systems_per_state?.optimized?.count;   /* eslint-disable-line camelcase */
     const newLine = '\n';
-    const bullet = '\u2022';
 
     const ioOccuranceTableData = [
```

Comment:
Typo
```suggestion
    const ioOccurrenceTableData = [
```

Reply:
fixed it in latest commit, thanks @r14chandra!